src/base/memory_monitor_sr.cpp

```diff
@@ -720,3 +831,81 @@ void mmon_aggregate_tran_info (int tran_count, MMON_TRAN_INFO &info)
 
   mmon_Gl->aggregate_tran_info (tran_count, info);
 }
+
+MMON_STAT_ID mmon_set_tracking_tag (MMON_STAT_ID new_tag)
+{
+  MMON_STAT_ID prev_tag = MMON_STAT_LAST;
+  THREAD_ENTRY *cur_thread_p;
+
+  if (mmon_Gl != nullptr)
+    {
+      cur_thread_p = thread_get_thread_entry_info ();
```

Comment:
What happens `cur_thread_p` is nullptr? It can be null though I'm not sure current usages run into this.

Reply:
We do not assume that thread_p == NULL. Since mmon_Gl is initialized after the initialization of the thread_manager, it guarantees accessibility to all threads managed by the thread_manager. If another developer is creating threads without using THREAD_ENTRY, the issue is whether these threads should be incorporated into the thread_manager. Considering this should be a separate concern.